telluric/collections.py

```diff
@@ -179,15 +184,22 @@ def rasterize(self, dest_resolution, polygonize_width=0, crs=WEB_MERCATOR_CRS, f
                   if not feature.is_empty]
 
         if bounds is None:
-            bounds = self.convex_hull.get_shape(crs)
+            bounds = self.envelope
```

Comment:
doesn't  this change ends up with a larger image array?

Reply:
no, because in the end the envelope will be used anyway (via the `bounds` property) and this one is faster to compute